prow/artifactuploader/BUILD.bazel

```diff
@@ -3,7 +3,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = ["controller.go"],
-    importpath = "k8s.io/test-infra/prow/artifact-uploader",
+    importpath = "k8s.io/test-infra/prow/artifactuploader",
```

Comment:
This commit will break downstream consumers who build this -- maybe let's just drop the commit and live with a hyphen?

Reply:
That sounds reasonable